proto/process/agent.proto

```diff
@@ -891,3 +894,10 @@ enum HTTPResponseStatus {
     ClientErr = 3;	// 4XX
     ServerErr = 4;	// 5XX
 }
+
+message NetworkTracerTelemetry {
+	int64 runtimeCompilationEnabled = 1;
+	int64 runtimeCompilationSuccess = 2;
+	int64 runtimeCompilationFailure = 3;
```

Comment:
is there an enum we can use for this to detect different failure modes in the future?